README.md

```diff
@@ -1,90 +1,130 @@
+* [Interested in contributing?]()
+* [MEV Roasts recordings]()
+* [List of Resources]()
+* [Navigate Flashbots' Github]()
 
+----
 ## Flashbots Alpha
 
-Flashbots Alpha is an early stage implementation of a direct communication channel between miners and Ethereum users that would like to communicate their preferred transaction order within a block.
+On Jan 6th 2021, we entered the Flashbots Alpha phase by opening up a proof-of-concept communication channel between traders and miners. This proof-of-concept is made of two components: MEV-Geth, a slightly modified fork of the Ethereum Geth client, and MEV-Relay, a miner relayer.
 
-Instead of broadcasting to the peer-to-peer network and competing with all Ethereum users, Flashbots users bid against each other in providing a reward directly to miners through a seal-bid auction. DeFi traders such as liquidators and arbitrageurs, can use Flashbots to bid for transaction inclusion without paying any gas fee for failed transactions, nor do they need to worry about leaking strategies before their trade was mined on-chain. 
+**Alpha flow:**
+* A trader sends a Flashbots bundle to MEV-Relay. The bundle contains:
+  * one or several transactions that can be the trader's and/or pending transactions in the mempool
+  * a sealed tip that is paid to the miner
+* MEV-Relay receives this bundle and sends it to all whitelisted MEV-Geth miners
+* Miners receive Flashbots bundles through MEV-Geth. 
+  * there can only be a single bundle per block
+  * the Flashbots bundle will always be at the top slot of the block by design of the system.
+* MEV-Geth picks which bundle to choose for each block.
+  * MEV-Geth will pick the most profitable bundle for the miner.
+  * MEV-Geth will compare the block that includes the bundle vs a vanilla Geth block, and will default back to the vanilla Geth block in case it is more profitable.
+* Only when the trader's bundle is included in a block do they pay the tip associated to their bundle.
+  * a bundle that isn't included costs the sender nothing (ie. no gas fees are paid for failed transactions)
 
-Miners and/or mining pools can receive MEV revenue, which is expected to be higher than revenue from mining regular blocks, for including the most profitable transaction bundles submitted by Flashbots users.
+This infrastructure allows:
+* traders to bypass the Ethereum mempool and avoid their strategy leaking before it is mined on-chain. (ie. being frontrun by generalized frontrunners)
+* traders to save money from avoiding to pay gas fees for failed transactions.
+* miners to receive additional revenue in the form of the bundle tip, in exchange for including the most profitable bundle in the block they mined.
 
-Flashbots aim to provide a "direct fast lane" for transaction inclusion with an efficient, fair and sustainable MEV auction mechanism, potentially reducing network congestion and lowering transaction fees for regular Ethereum users.
+Interested in learning more?
```

Comment:
perhaps use "subsection" as above? **Interested in learning more?**

Reply:
not sure what you mean here, do you mean bullet points to the links?

Reply:
I meant using the same level in the hierarchy as **Alpha flow:** above

Reply:
I just tried it and think I like it better without it, not a lot of use to info hierarchy for 3 links at the moment. Lmk if you feel strongly about having bullet points instead :p